cmd/controller-manager/app/options/options.go

```diff
@@ -78,6 +80,8 @@ func (s *KubeSphereControllerManagerOptions) Flags() cliflag.NamedFlagSets {
 		"if not set, webhook server would look up the server key and certificate in"+
 		"{TempDir}/k8s-webhook-server/serving-certs")
 
+	fs.StringVar(&s.KubectlImage, "kubectl-image", s.KubectlImage, "Setup the image used by kubectl terminal pod")
```

Comment:
I think we'd better to set kubectlimage through config file, otherwise, if we ks-installer restarted, setting will be overrided.